docs/generated/packages/react-native/generators/component-story.json

```diff
@@ -15,7 +15,8 @@
         "description": "The project where to add the components.",
         "examples": ["shared-ui-component"],
         "$default": { "$source": "projectName", "index": 0 },
-        "x-prompt": "What's the name of the project where the component lives?"
+        "x-prompt": "What's the name of the project where the component lives?",
```

Comment:
Same here, `required` and `important` don't both need to be set. I won't add a comment for any other of these duplicates :D

Reply:
removed